sw/host/cryptotest/testvectors/parsers/nist_cavp_aes_kw_parser.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+"""Parser for converting NIST AES-KWP testvectors to JSON.
+
+"""
+
+import argparse
+import sys
+import json
+import jsonschema
+
+from cryptotest_util import parse_rsp, str_to_byte_array
+
+
+def parse_testcases(args) -> None:
+    raw_testcases = parse_rsp(args.src)
+    testcases = list()
+    test_case_id = 1
+    for test_key in raw_testcases.keys():
+        for test_vec in raw_testcases[test_key]:
```

Comment:
We'll have to change this code to remove the double loop to merge #21728, but that shouldn't block this PR.

Reply:
Locally I have already verified it by removing the nested loop to follow the new parsing logic. It works fine. I will create another PR to change this.